src/main/java/sirius/kernel/cache/ManagedCache.java

```diff
@@ -341,6 +342,11 @@ public void removeIf(@Nonnull Predicate<CacheEntry<K, V>> predicate) {
         return this;
     }
 
+    @Override
+    public CacheRemoverBuilder<K, V, CacheEntry<K, V>> addRemover(@Nonnull String discriminator) {
```

Comment:
also add a second method addValueBasedRemove or the like, which directly operates on the value of the cache entry (which is what we do in 95% of the time)